tests/installation/grub_test.pm

```diff
@@ -59,7 +59,7 @@ sub run {
     }
     workaround_type_encrypted_passphrase;
     # 60 due to rare slowness e.g. multipath poo#11908
-    assert_screen "grub2", 60;
+    assert_screen "grub2", 300;
```

Comment:
Hack, as somehow system boots to the iso after reboot. With increased timeout we start booting from hdd and everything works well.